internal/config/config.go

```diff
@@ -36,6 +36,7 @@ type Config struct {
 	Tracing       TracingConfig      `mapstructure:"tracing"`
 	Metrics       MetricsConfig      `mapstructure:"metrics"`
 	Database      DatabaseConfig     `mapstructure:"database"`
+	DatabaseQueue DatabaseConfig     `mapstructure:"database_queue"`
```

Comment:
This database configuration is explicitly part of the eventing system. Let's not make it a top-level configuration and instead put that inside of the `events` section, within the driver configuration.